edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -87,7 +87,7 @@ public async Task<bool> CloseAsync()
                 // wrapping this.client.CloseAsync in a try/catch
                 try
                 {
-                    await this.client.CloseAsync();
+                    await this.client.CloseAsync().TimeoutAfter(this.cloudConnectionHangingTimeout);
```

Comment:
Technically out of scope for this PR, but it looks like this value can be changed via the `CloudConnectionHangingTimeoutSecs` env var. Can we document this in the table of EdgeHub env vars in [EnvironmentVariables.md](https://github.com/Azure/iotedge/blob/release/1.4/doc/EnvironmentVariables.md)?

Reply:
I'll check with Bilal when it was first added.